src/index.css

```diff
@@ -0,0 +1,214 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    Lato,
+    'Gill Sans',
+    'Gill Sans MT',
+    Calibri,
+    'Trebuchet MS',
+    sans-serif;
```

Comment:
- As pointed out by the previous reviewer, the font do not yet conform to the [design guideline.](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) 
- When I inspect your page I get this font family returned:

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/204997947-0d896fd9-60c3-4373-b9e8-7cfbb072e155.png) | ![image](https://user-images.githubusercontent.com/27709832/204998141-5366a85e-de06-4103-9e29-a014f3cc5e65.png).

- Note that you could target the classes holding your navigation (control) with the Lato font family as well as the card holding the body content with the Gill sans font family.